sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/src/autorest.md

```diff
@@ -7,7 +7,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 azure-arm: true
 namespace: Azure.ResourceManager.DeviceUpdate
 require: https://github.com/Azure/azure-rest-api-specs/blob/32143b0f5f230ee2601e3c5d1990188666a5058d/specification/deviceupdate/resource-manager/readme.md
-tag: package-2020-03-01-preview
+tag: package-2022-04-01-preview
 clear-output-folder: true
```

Comment:
please run `eng/script/Export-API.ps1 deviceupdate` to solve the ci failure

Reply:
I did but looks it is still failing with ChangeLog error, however it is present for the 1.0.0-beta.5 version, why is it failing still?

Error : No entry for version [1.0.0-beta.5] found in the ChangeLog [D:\a\_work\1\s\sdk\deviceupdate\Azure.ResourceManager.DeviceUpdate\CHANGELOG.md].